tests.py

```diff
@@ -654,67 +673,68 @@ def test_make_pr_link(github_api):
     )
 
 
-def test_obtain_branches_to_delete(mock_stash_api, github_api):
-    from_branch = "fb-ASIM-81-network"
-    mock_stash_api["PROJ-A"]["repo1"]["pull_requests"] = [
-        dict(
-            id="10",
-            fromRef=dict(id="refs/heads/" + from_branch),
-            toRef=dict(id="refs/heads/target_branch"),
-            displayId=from_branch,
-            links=make_link("url.com/for/10"),
-            version="10",
-            state="DECLINED",
-        )
-    ]
-    mock_stash_api["PROJ-A"]["repo1"]["pull_request"] = {"10": DummyPullRequest(True)}
-    mock_stash_api["PROJ-A"]["repo1"]["commits"] = {
-        ("refs/heads/" + from_branch, "refs/heads/target_branch"): ["A", "B"]
-    }
-
+@pytest.mark.parametrize(
+    "branch_name, expected_messages",
+    [
+        (
+            "fb-ASIM-81-network",
+            (
+                "Stash: repo1 (commit id *2a0ae67e039099e300ec972e22c281af19f4ac9d*) \n"
+                "Stash: repo3 (commit id *a938dfdfbaa1f25ccbc39e16060f73c44e5ef0dd*) *has PR*\n"
+                "GitHub: deps (commit id *e685cbba9aab1683a4a504582b4e30af36cdfddb*)"
+            ),
+        ),
+        (
+            "fb-branch-only-in-github",
+            "GitHub: deps (commit id *e685cbba9aab1683a4a504582b4e30af36cdfddb*)",
+        ),
+    ],
+)
+def test_obtain_branches_to_delete(
+    mock_stash_api, github_api, branch_name, expected_messages
+):
     branches_to_delete = list()
     stash_api = StashAPI(
         "https://myserver.com/stash", username="fry", password="PASSWORD123"
     )
-    github_api.repos["esss"]["deps"].branches.append(GitBranch(from_branch))
+    github_api.repos["esss"]["deps"].branches.append(GitBranch(branch_name))
     lines = list(
         obtain_branches_to_delete(
             stash_api,
             github_api,
             ["PROJ-A", "PROJ-B"],
             ["esss"],
-            from_branch,
+            branch_name,
             branches_to_delete,
         )
     )
-    assert (
-        "\n".join(lines) == "Found branch `fb-ASIM-81-network` in these repositories:\n"
-        "Stash: repo1 -> (commit id *2a0ae67e039099e300ec972e22c281af19f4ac9d*) *has PR*\n"
-        "Stash: repo3 -> (commit id *2a0ae67e039099e300ec972e22c281af19f4ac9d*) *has PR*\n"
-        "GitHub: deps -> (commit id *2a0ae67e039099e300ec972e22c281af19f4ac9d*) \n"
-        "*To confirm to delete this branches please _repeate_ the command*"
+    assert "\n".join(lines) == (
+        f"Found branch `{branch_name}` in these repositories:\n{expected_messages} \n*To confirm to delete this branches please _repeate_ the command*"
     )
-    assert len(branches_to_delete) == 3
+    assert len(branches_to_delete) == len(expected_messages.split("\n"))
 
 
 def test_delete_branches(mock_stash_api, github_api):
```

Comment:
Please also parametrize this test so we test: deleting GH + Stash branches, only Stash branches, only GH branches. 👍